addons/crm/models/res_partner.py

```diff
@@ -64,6 +64,11 @@ def action_view_opportunity(self):
         '''
         action = self.env['ir.actions.act_window']._for_xml_id('crm.crm_lead_opportunities')
         action['context'] = {}
+
+        list_view = [(state, view) for state, view in action['views'] if view == 'list']
```

Comment:
Not sure, but couldn't we accomplish this directly with `self.env.ref`?

Reply:
Hello @ibo-odoo 
Yes, we can use `self.env.ref` to get a list view and display it. However, in this case, we want to display all the views by simply reordering them. Since we need to use a loop to retrieve all the views anyway, so decided to use it twice for consistency.

Please correct me if I have missed something.

Thanks :blush: